common/views/components/EventBookingButton/EventBookingButton.js

```diff
@@ -0,0 +1,70 @@
+// @Flow
+
+import {spacing, font} from '../../../utils/classnames';
+import {Fragment} from 'react';
+import Icon from '../Icon/Icon';
+import ButtonButton from '../Buttons/ButtonButton/ButtonButton';
+
+type Props = {|
+  event: Event, // TODO
+|}
+
+function getButtonMarkup(event) {
+  if (!event.eventInfo.eventbriteId) return;
+
+  if (event.eventInfo.isCompletelySoldOut) {
+    return (
+      <ButtonButton
+        text='Fully booked'
+        icon='ticketAvailable'
+        extraClasses={`${font({s: 'HNM5'})} btn--full-width-s`}
+      />
+    );
+  } else {
+    return (
+      <div className="js-eventbrite-ticket-button" data-eventbrite-ticket-id={event.eventInfo.eventbriteId}>
+        <a className={`flex-inline flex--v-center flex--h-center btn btn--full-width-s ${font({s: 'HNM4'})}`}
+          href={`https://www.eventbrite.com/e/${event.eventInfo.eventbriteId}/`}>
+          <span><Icon name='ticketAvailable' /></span>
+          <span className="js-eventbrite-ticket-button-text">Book free tickets</span>
+        </a>
+      </div>
+    );
+  }
+}
+
+function getBookingEnquiryMarkup(event) {
+  if (!event.bookingEnquiryTeam) return;
+
+  if (event.eventInfo.isCompletelySoldOut) {
```

Comment:
See the slack message, but basically this would be better on the root object - if we use it in multiple places, let's stick it on the model.